packages/adblocker-playwright-example/package.json

```diff
@@ -53,7 +53,7 @@
   "dependencies": {
     "@ghostery/adblocker-playwright": "^2.8.0",
     "cross-fetch": "^4.0.0",
-    "playwright": "^1.38.0"
+    "playwright": "^1.52.0"
```

Comment:
chore: I think we can match playwright version with `adblocker-extended-selectors`

Reply:
yarn resolves both to the same version